oak-run/src/main/java/org/apache/jackrabbit/oak/run/DataStoreCommand.java

```diff
@@ -155,21 +168,81 @@ private static boolean checkParameters(DataStoreOptions dataStoreOpts, Options o
     private void execute(NodeStoreFixture fixture,  DataStoreOptions dataStoreOpts, Options opts, Closer closer)
         throws Exception {
 
+        final BlobStoreOptions optionBean = opts.getOptionBean(BlobStoreOptions.class);
         try (Closer metricsCloser = Utils.createCloserWithShutdownHook()) {
             MetricsExporterFixture metricsExporterFixture =
                 MetricsExporterFixtureProvider.create(dataStoreOpts, fixture.getWhiteboard());
             metricsCloser.register(metricsExporterFixture);
 
-            MarkSweepGarbageCollector collector = getCollector(fixture, dataStoreOpts, opts, closer);
-            if (dataStoreOpts.checkConsistency()) {
-                long missing = collector.checkConsistency();
-                log.warn("Found {} missing blobs", missing);
+            if (dataStoreOpts.dumpRefs()) {
+                final File referencesTemp = File.createTempFile("traverseref",null, new File(opts.getTempDirectory()));
+                final BufferedWriter writer = Files.newWriter(referencesTemp, Charsets.UTF_8);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unnecessary use of fully qualified name 'Charsets.UTF_8' due to existing static import 'com.google.common.base.Charsets.UTF_8'](https://app.codacy.com/gh/oak-indexing/jackrabbit-oak/pullRequest?prid=5079914)